timemachine/cpp/src/chiral_atom_restraint.hpp

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#include "potential.hpp"
+#include <vector>
+
+namespace timemachine {
+
+template <typename RealType> class ChiralAtomRestraint : public Potential {
+
+private:
+    int *d_idxs_;
+
+    const int R_;
+
+public:
+    ChiralAtomRestraint(const std::vector<int> &idxs // [R, 4]
+    );
```

Comment:
nit: weird formatting

Reply:
Hm - I think it looked a little different before the pre-commit hook automatically formatted this. Maybe the comment throws it off a little?

Reply:
Ah, sorry, I think it's fine actually. I didn't appreciate that the comment is supposed to be paired with the argument versus the whole declaration.